source/NVDAHelper.py

```diff
@@ -647,13 +647,24 @@ def nvdaControllerInternal_typedCharacterNotify(ch):
 
 
 @WINFUNCTYPE(c_long, c_int, c_int, c_bool)
-def nvdaControllerInternal_vbufChangeNotify(rootDocHandle: int, rootID: int, selectionChanged: bool) -> int:
+def _nvdaControllerInternal_vbufChangeNotify(rootDocHandle: int, rootID: int, selectionChanged: bool) -> int:
 	import virtualBuffers
 
 	virtualBuffers.VirtualBuffer.changeNotify(rootDocHandle, rootID, selectionChanged)
 	return 0
 
 
+def nvdaControllerInternal_vbufChangeNotify(
```

Comment:
can you please use our deprecations format and add a change log entry?

https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/deprecations.md